artstor/manifest.json

```diff
@@ -3,6 +3,7 @@
   "status": "beta",
   "domains": [
     "library.artstor.org"
```

Comment:
Json elements need to be separated by commas.

Reply:
Done